README.md

```diff
@@ -66,7 +70,7 @@ Select 1:
 ```
 [] Selects all li's which are directly inside a ul of class dropdown (children)
 [] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
-[] Selects all ul's of class dropdown, as well as the children elements that are li's
+[x] Selects all ul's of class dropdown, as well as the children elements that are li's
```

Comment:
We were actually going for the first choice, but the way this is worded you're technically correct so I'll give it to you!
:+1: